From d3ab7ba386f17a19de59391264d7c5f06c160991 Mon Sep 17 00:00:00 2001
From: R?mi Verschelde <rverschelde@gmail.com>
Date: Tue, 13 Nov 2018 12:00:34 +0100
Subject: [PATCH 1/6] CMake: Fix finding OIS 1.4+

* With version 1.4, upstream OIS changed the location of their
  system-installed headers from /usr/include/OIS to /usr/include/ois.
---
 application_templates/Ogre.cpp | 2 +-
 cmake/FindOIS.cmake            | 8 +-------
 2 files changed, 2 insertions(+), 8 deletions(-)

diff --git a/application_templates/Ogre.cpp b/application_templates/Ogre.cpp
index 5ee466d63..c2b8741f6 100644
--- a/application_templates/Ogre.cpp
+++ b/application_templates/Ogre.cpp
@@ -333,4 +333,4 @@ int main(int argc, char *argv[]){
 	}
 
 	return 0;
-}
\ No newline at end of file
+}
diff --git a/cmake/FindOIS.cmake b/cmake/FindOIS.cmake
index 7a4d015b4..14731a153 100644
--- a/cmake/FindOIS.cmake
+++ b/cmake/FindOIS.cmake
@@ -15,13 +15,7 @@ set(OIS_PREFIX_PATH
 
 find_path(OIS_H_PATH NAMES OIS.h
           PATHS ${OIS_PREFIX_PATH} 
-          PATH_SUFFIXES OIS include includes)
-        
-if(NOT OIS_H_PATH)
-    find_path(OIS_H_PATH NAMES OIS.h
-              PATHS ${OIS_PREFIX_PATH} 
-              PATH_SUFFIXES OIS include include/OIS)
-endif()
+          PATH_SUFFIXES ois OIS include includes include/ois include/OIS)
         
 find_library(OIS_LIB NAMES OIS libOIS
              PATHS ${OIS_PREFIX_PATH}
-- 
2.26.2

